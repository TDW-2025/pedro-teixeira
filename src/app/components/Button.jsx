// src/components/Button.jsx

export const Button = ({ label }) => {
  return <button>{label}</button>;
};
