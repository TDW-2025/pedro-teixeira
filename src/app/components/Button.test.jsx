// src/components/Button.test.jsx
import { render } from "@testing-library/react";
import { Button } from "./Button";

describe("Button Component", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<Button label="Click me" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
