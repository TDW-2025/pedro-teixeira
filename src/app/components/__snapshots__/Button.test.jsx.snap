// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Button Component matches snapshot 1`] = `
<DocumentFragment>
  <button>
    Click me
  </button>
</DocumentFragment>
`;
